contrib/builder/deb/aarch64/generate.sh

```diff
@@ -97,18 +99,31 @@ for version in "${versions[@]}"; do
 			;;
 	esac
 
-	echo "# Install Go" >> "$version/Dockerfile"
-	echo "# aarch64 doesn't have official go binaries, so use the version of go installed from" >> "$version/Dockerfile"
-	echo "# the image to build go from source." >> "$version/Dockerfile"
+	case "$suite" in
+		jessie)
+			echo "# Install Go (hack: for now use pre-compiled tarball from Hypriot)" >> "$version/Dockerfile"
```

Comment:
Currently there is NO installable golang package for Debian/Jessie in arm64 available. That's exactly the reason why I decided to use a pre-compiled tarball. I did asked the Go team a year before to deliver arm64 tarballs but they don't want it.